bridge/jni/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "dsnp-graph-sdk-jni"
```

Comment:
nit: Let's standardize our crate naming; currently we have some `dsnp-graph-*` and some `dsnp-graph-sdk-*`. My preference is for `dsnp-graph-*` (without SDK), but as long as we're consistent.

Reply:
let's talk about this in next graph-sdk meeting